setup.py

```diff
@@ -35,7 +35,8 @@
     "nose",
     "six",
     "Cython",
-    "numpy>=1.9.0",
+    # Numpy version of higher than 1.14.5 causes libgcc_s.so.1 error.
```

Comment:
I would prefer if you use a `numpy>=1.9.0<=1.14.5` to keep it more flexible. Also, is this only online or also in a fresh, local conda environment.